src/main/webapp/app/assessment-instructions/expandable-problem-statement/expandable-problem-statement.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { ProgrammingExercise } from 'app/entities/programming-exercise';
+import { SafeHtml } from '@angular/platform-browser';
+import { ArtemisMarkdown } from 'app/components/util/markdown.service';
+import { Exercise, ExerciseType } from 'app/entities/exercise';
+
+@Component({
+    selector: 'jhi-expandable-problem-statement',
+    templateUrl: './expandable-problem-statement.component.html',
+    styleUrls: ['../assessment-instructions.scss'],
```

Comment:
Do we need exactly the same styles for both places? If not I would suggest to split it into two files

Reply:
Yes, we do need the same style :)